src/Network/Network/Models/PSEffectiveRoute.cs

```diff
@@ -25,6 +25,10 @@ public class PSEffectiveRoute
         [Ps1Xml(Target = ViewControl.Table)]
         public string Name { get; set; }
 
+        [JsonProperty(Order = 1)]
```

Comment:
How is this going to be populated?  Does it need an update to the Map for this type?

Reply:
No update to map needed